src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandHandler.cs

```diff
@@ -386,8 +386,9 @@ private async Task AddCommPkgsAsync(Invitation invitation, IList<string> commPkg
             var initialCommPkg = commPkgDetailsList.FirstOrDefault();
             if (initialCommPkg != null)
             {
-                var initialSystem = initialCommPkg.System;
-                if (commPkgDetailsList.Any(commPkg => commPkg.System != initialSystem))
+                var initialSystem = initialCommPkg.System.Substring(0, initialCommPkg.System.LastIndexOf('|'));
```

Comment:
make a get-property in CommPkg named SubSystem returning .System.LastIndexOf('|')
Do it for all places where you have this logic